### DISCUSSION TITLE
when building with wayland it doesnt work and i get this error

### DISCUSSION THREAD

**Discussion by itzRealDuck** [elkowar/eww#726] [Q&A]
so i was just building eww but it gave this error "   Compiling gtk-layer-shell-sys v0.4.4
The following warnings were emitted during compilation:

warning: `"pkg-config" "--libs" "--cflags" "gtk-layer-shell-0" "gtk-layer-shell-0 >= 0.1"` did not exit successfully: exit status: 1

error: failed to run custom build command for `gtk-layer-shell-sys v0.4.4`

Caused by:
  process didn't exit successfully: `/home/itzrealduck/eww/target/release/build/gtk-layer-shell-sys-1c5ffceb9b7e6a98/build-script-build` (exit status: 1)
  --- stdout
  cargo:rerun-if-env-changed=GTK_LAYER_SHELL_0_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-unknown-linux-gnu
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_unknown_linux_gnu
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR
  cargo:warning=`"pkg-config" "--libs" "--cflags" "gtk-layer-shell-0" "gtk-layer-shell-0 >= 0.1"` did not exit successfully: exit status: 1
  error: could not find system library 'gtk-layer-shell-0' required by the 'gtk-layer-shell-sys' crate

  --- stderr
  Package gtk-layer-shell-0 was not found in the pkg-config search path.
  Perhaps you should add the directory containing `gtk-layer-shell-0.pc'
  to the PKG_CONFIG_PATH environment variable
  Package 'gtk-layer-shell-0', required by 'virtual:world', not found
  Package 'gtk-layer-shell-0', required by 'virtual:world', not found" so i installed the aur version of eww and made a execcutable and a eww directory in .config and then eww open example in the command line but it didnt do anything i thought i need to build it so please help

**Comment by elkowar** [comment]
Reading the error message, I have this weird hunch that it might be related to you missing gtk-layer-shell..
If you're on arch, Iirc there's a eww-Wayland package that has that dependency, otherwise just install it yourself
